mailpoet/lib/WooCommerce/MultichannelMarketing/MPMarketingChannel.php

```diff
@@ -157,7 +157,7 @@ protected function generateCampaignTypes(): array {
           'Send a newsletter with images, buttons, dividers, and social bookmarks. Or, just send a basic text email.',
           'mailpoet',
         ),
-        admin_url('admin.php?page=' . Menu::EMAILS_PAGE_SLUG . '#/new/standard'),
+        admin_url('admin.php?page=' . Menu::EMAILS_PAGE_SLUG . '#/new/standard/loadedvia=woo_multichannel_dashboard'),
```

Comment:
For this and the post notification, is there a reason you want to have the `loadedvia` as part of the hash instead of using a standard search param? It seems like using a search param in all cases would be simpler and feels more idiomatic to me.

Reply:
I appreciate your suggestion.
I tried it previously, but it wasn’t working.
It may have something to do with the way the routes are set [here](https://github.com/mailpoet/mailpoet/blob/9e8c26850dd346458cbb427ddaf10201e651307a/mailpoet/assets/js/src/newsletters/newsletters.jsx#L184-L242).
Converting the `loadedvia` part to search param instead of hash prevents the navigation from working correctly.
Here is a quick video: 
https://github.com/mailpoet/mailpoet/assets/30554163/ab60f9bc-605f-4149-aba7-f57cc64c6007

Reply:
Update

I implemented the feature here: https://github.com/mailpoet/mailpoet/pull/5331/commits/6018682551626d264fbcc9861b478c43c0572369.

I could do this by moving the `loadedvia` part to the beginning of the URL. For example, `/wp-admin/admin.php?loadedvia=woo_multichannel_dashboard&page=mailpoet-newsletters#/new/notification`.

I don't know if there's going to be any side-effects of using `/wp-admin/admin.php?loadedvia=woo_multichannel_dashboard&page=mailpoet-newsletters#/new/notification.` instead of `/wp-admin/admin.php?page=mailpoet-newsletters#/new/notification`.

Do let me know what you think.

Reply:
Grateful for your insight. I decided to adopt your suggestion here https://github.com/mailpoet/mailpoet/pull/5331/commits/f593fb501027694a5db80fa5001d370db9cc6296.